front/app/modules/commercial/insights/utils/canvasShapes.ts

```diff
@@ -0,0 +1,97 @@
+// utils
+import { colors } from 'utils/styleUtils';
+
+// typings
+import { IInsightsNetworkNodeMeta } from 'modules/commercial/insights/admin/containers/Insights/Details/Network';
+import { NodeObject } from 'react-force-graph-2d';
+
+type Node = NodeObject & IInsightsNetworkNodeMeta;
+
+function hideIconBoxParams(node: Node): [number, number, number, number] {
```

Comment:
Really love that you took those out of the component!

Reply:
yes it really makes sense to move all this drawing to a separated file